tests/storage/export.go

```diff
@@ -1271,12 +1271,10 @@ var _ = SIGDescribe("Export", func() {
 		Expect(*snapshot.Status.SourceUID).To(Equal(vm.UID))
 
 		contentName := *snapshot.Status.VirtualMachineSnapshotContentName
-		Expect(snapshot.Status.Indications).To(BeEmpty())
 		content, err := virtClient.VirtualMachineSnapshotContent(vm.Namespace).Get(context.Background(), contentName, metav1.GetOptions{})
 		Expect(err).ToNot(HaveOccurred())
 
 		Expect(*content.Spec.VirtualMachineSnapshotName).To(Equal(snapshot.Name))
-		Expect(content.Spec.Source.VirtualMachine.Spec).To(Equal(vm.Spec))
```

Comment:
same as above

Reply:
Turns out to also not always be to be equal, again not really that import that they match exactly for this test.